monitor/grafana/zeebe.json

```diff
@@ -4912,9 +5035,10 @@
           "linewidth": 1,
           "nullPointMode": "null",
           "options": {
-            "dataLinks": []
+            "alertThreshold": true
```

Comment:
:thought_balloon:  Confused by this, I hope it's harmless :smile:

Reply:
Yeah this is added per default now